README.md

```diff
@@ -89,6 +97,25 @@ return assert.isRejected(promise, Error, "optional message");
 return assert.isRejected(promise, /error message matcher/, "optional message");
 ```
 
+### Multiple Promise assertions
```

Comment:
You are correct, I see it now.

Sorry, that is under "non-promise-friendly test runners". "Some test runners (e.g. Jasmine, QUnit, or tap/tape) do not have the ability to use the returned promise to signal asynchronous test completion"

However, I'm using Mocha, Chai, Chai-as-Required, and it's perfectly friendly to using `return`. The thing is, `return` doesn't work in the scenario of multiples regardless of friendliness towards promise returns. At very least, would it be acceptable to pull the current example out of the heading "non-promise-friendly test runners" and put it into this heading instead?